multinode_integration_tests/src/masq_node.rs

```diff
@@ -157,13 +149,13 @@ impl NodeReference {
             vec![]
         } else {
             String::from(slice)
-                .split(',')
+                .split(NodeAddr::PORTS_SEPARATOR)
                 .map(|x| x.parse::<i64>().unwrap_or(-1))
                 .collect()
         };
         if port_list_numbers.contains(&-1) {
             return Err(format!(
-                "The port list must be a comma-separated sequence of valid numbers, not '{}'",
+                "The port list must be a semicolon-separated sequence of valid numbers, not '{}'",
```

Comment:
Did you decide against `/` separated port number? It looks like there's a test below `strip_ports_works_multiple_ports` that addresses `/`. This description of the requirement may not match the implementation, correct?